src/DynamoRevit/DynamoRevit.cs

```diff
@@ -607,8 +607,7 @@ private static RevitDynamoModel InitializeCoreModel(DynamoRevitCommandData comma
                 AuthProvider = new RevitOAuth2Provider(new DispatcherSynchronizationContext(Dispatcher.CurrentDispatcher)),
                 ExternalCommandData = commandData,
                 UpdateManager = revitUpdateManager,
-                ProcessMode = isAutomationMode ? TaskProcessMode.Synchronous : TaskProcessMode.Asynchronous,
-                Preferences = PreferenceSettings.Instance
```

Comment:
> Does removing this not bring back other issues with Preferences on Dynamo Revit startup?

No because we are returning the responsibility of management/creation of the preferences to the DynamoModel here https://github.com/DynamoDS/Dynamo/pull/14427, we need both to make them work properly.

Reply:
This is reverting the code back to https://github.com/DynamoDS/DynamoRevit/blob/RC2.18.1_Revit2024.1/src/DynamoRevit/DynamoRevit.cs#L476. I think it was our mistake that these changes were not needed in the first place